Refit/ApiResponse.cs

```diff
@@ -27,7 +27,7 @@ public class RefitResponse<T> : IRefitResponse<T>
             Content = content;
         }
 
-        internal RefitResponse(HttpResponseMessage response,
+        internal ApiResponse(HttpResponseMessage response,
```

Comment:
Do we need this here? Can we remove this overload and do the cast where needed externally?

Reply:
I removed the internal costructor and the cast as is not needed.